templates/shared/_pagination.html

```diff
@@ -1,89 +1,3 @@
 <div class="u-fixed-width">
-  {% if page and total_pages and total_pages > 1 %}
```

Comment:
Is this change intentional? If so, then we don't need this file as the new version of it is empty.